app/views/payments/_form.html.haml

```diff
@@ -0,0 +1,77 @@
+- invoice_amount = @invoice.total_amount
+- paid_amount = @invoice.total_amount_payments
+- date_of_payment = date_of_payment_form(@payment)
```

Comment:
This local variable is used only once. Can we directly use the expression in place?